### PR TITLE
Fix a few typos

### PR DESCRIPTION
**Pull request by kinow**
Hi,

@hjoliver found a typo in #3006 , and I had found two other typos while trying to work on the #3003 . So decided to prepare a separate pull request with these three typos, plus whatever I could find in 30 minutes tonight before our meeting.

As always, not a native speaker, so there may be some typos in the fix for the typos :grimacing: Hoping I did not break a functional test in Travis-CI.

Cheers
Bruno

### CODE REVIEW

**Comment by kinow** [comment]
Oh, and there were a few typos in the `cylc-profile-battery` and in the profile experiments folder, but I did not include them to make it easier to merge #3006 :+1:

File: etc/examples/inherit/multi/one/suite.rc [hunk 1]
```diff
@@ -41,7 +41,7 @@
 [visualization]
     # NOTE ON VISUALIZATION AND MULTIPLE INHERITANCE: overlapping
     # family groups can have overlapping attributes, so long as 
-    # non-conflictling attributes are used to style each group. Below,
+    # non-conflicting attributes are used to style each group. Below,
```
Comment (hjoliver) [inline etc/examples/inherit/multi/one/suite.rc]: He heh.  This is my favourite one :grin:

File: lib/parsec/validate.py [hunk 2]
```diff
@@ -276,11 +276,11 @@ def coerce_spaceless_str_list(cls, value, keys):
...
 
     @classmethod
-    def coerce_asbolute_host_list(cls, value, keys):
+    def coerce_absolute_host_list(cls, value, keys):
         """Do not permit self reference in host names.
 
         Example:
-            >>> ParsecValidator.coerce_asbolute_host_list(
+            >>> ParsecValidator.coerce_absolute_host_list(
             ...     'foo, bar, 127.0.0.1:8080, baz', ['pub'])
```
Comment (hjoliver) [inline lib/parsec/validate.py]: Wow, mutually consistent crazy typos, brilliant! :grin:

File: tests/triggering/17-suicide-multi.t [hunk 3]
```diff
@@ -15,7 +15,7 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #-------------------------------------------------------------------------------
-# Test "or" outputs from same task triggerting suicide triggering
```
Comment (hjoliver) [inline tests/triggering/17-suicide-multi.t]: Ooh, that's a good one too!

**Comment by hjoliver** [comment]
I suppose this should go to 7.8.x too.

**Pull request merged**